trigger.md

```diff
@@ -0,0 +1,80 @@
+# Finite Triggered Acquisition
+Acquire (`acquire-imaging` [on PyPI](https://pypi.org/project/acquire-imaging/)) is a Python package providing a multi-camera video streaming library focused on performant microscopy, with support for up to two simultaneous, independent, video streams.
+
+This tutorial shows an example of setting up triggered acquisition of a finite number of frames with one of Acquire's supported devices and saving the data to a Zarr file.
+
+## Initialize Acquisitoin 
+
+To start, we'll import `Acquire` and create an acquisition `Runtime` object, which initializes the driver adaptors needed for the supported cameras.
+```
+import acquire
+runtime = acquire.Runtime()
+```
+
+## Configure Camera
+
+All camera settings can be captured by an instance of the `Properties` class, which will be associated will a given camera acquisition. The settings can be stored in a dictionary (e.g: `Properties.dict()`). These settings can be saved to a json files to be subsequently loaded, (e.g. ` Properties(**json.load('acquire.json'))` ). 
```

Comment:
```suggestion
All camera settings can be captured by an instance of the `Properties` class, which will be associated will a given camera acquisition. The settings can be stored in a dictionary (e.g: `Properties.dict()`). These settings can be saved to a JSON file to be subsequently loaded, (e.g. ` Properties(**json.load('acquire.json'))` ). 
```
It might be worth giving an example how this is done, e.g.

```python
props = runtime.get_configuration()

import json
with open("/path/to/acquire.json", "w") as f:
    json.dump(props.dict(), f)
```
because it's not supported by the API.